hd/lang/lex_utf8.txt

```diff
@@ -10,20 +10,6 @@ fr: Tapez à l'identique le nom de la généalogie que vous voulez utiliser :
 en: list of available genealogies (use gwsetup)
 fr: listes des généalogies disponibles (utilise gwsetup)
 
-    about (short date)
```

Comment:
Create a simple, clear PR that says e.g.: Clean lex_utf8 with a simple commit and an up to date file. And then you keep it up to date when you modify it for your needs